transport/src/main/java/io/netty/channel/AbstractChannel.java

```diff
@@ -810,6 +822,10 @@ public final void flush() {
                 return;
             }
 
+            flush(outboundBuffer);
```

Comment:
So does this mean that explicit calls from the application will also flush?

I think if a channel is configured to autoflush, it should just ignore requests from the application to flush. This will greatly simplify the migration path from applications that are doing their own flushing today.